src/api/services/usersService.ts

```diff
@@ -20,10 +20,15 @@ export const enrichUserWithIsApprover = async (user: User) => {
   };
 };
 
-export const createUserInPortal = async (user: Omit<UserDTO, 'id' | 'acceptedTerms'>) => {
+export const createUserInPortal = async (
```

Comment:
I think it's worth throwing a TODO: in here for the follow-up ticket - this is used for the /invite handler, but right now if an existing user is invited it'll find it and return it without adding the new participant.